pytest_tests/tests/container/test_container.py

```diff
@@ -32,6 +39,87 @@ def object_should_be_gc_marked(neofs_env: NeoFSEnv, node: StorageNode, cid: str,
 
 
 class TestContainer(TestNeofsBase):
+    def _create_wallet_for_account(self, account: neo3_account.Account, prefix: str) -> str:
+        wallet_path = os.path.join(get_assets_dir_path(), f"{prefix}-{str(uuid.uuid4())}.json")
+        wallet = neo3_wallet.Wallet()
+        wallet.account_add(account)
+        with open(wallet_path, "w") as out:
+            json.dump(wallet.to_json(), out)
+        return wallet_path
+
+    def _create_multi_account_wallet(self, accounts: list[neo3_account.Account], prefix: str) -> str:
```

Comment:
For this particular test I'd prefer two separate wallets although it's not critical.